<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassportWgt</class>
 <widget class="QWidget" name="PassportWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Информация об устройстве:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Код производителя (кирилица загл. 4 символа):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="kodPrlineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Код устройства (формат xxxxxx.xxx):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="kodUstrlineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Номер иcполнения(0-если нет, макс. 100):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="NomerIsplineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Доп. номер исполнения(0-если нет, макс. 100):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dopNomerIsplineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Заводской номер(цифровой, 0-нет номера):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="zavNomlineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Дополнительное обозначения(макс. 16 символов):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dopOblineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Заводской номер(макс. 16 символов):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="zavNomStrlineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Дата изготовления:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dataIzgotdateTimeEdit"/>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string> Информация о ПО:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Общая информация о ПО(тип, частота):</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Тип:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Частота:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="chastProclineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Дата файла:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dataFiledateTimeEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Версия ПО(0-нет информации о версии ПО):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="versPolineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Контрольная сумма:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="crclineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Ревизия процессора/плис(0-ревизия не доступна):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="revProclineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Дата компиляции(0-дата недотсупна):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dataCompdateTimeEdit"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Отправить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
